docs/specs/config.md

```diff
@@ -91,33 +87,26 @@ dependencies:
   samples-mediaservices-encoderstandard: https://github.com/Azure-Samples/media-services-dotnet-on-demand-encoding-with-media-encoder-standard
   samples-durable-functions: https://github.com/Azure/azure-functions-durable-extension
   samples-luis: https://github.com/Microsoft/Luis-Samples
-  cli_scripts: 
-    url: https://github.com/Azure/azure-docs-cli-python-samples
-    branch_mapping: 
-      release-build-mysql: release-build
-      release-build-postgresql: release-build
-      release-build-stellar: release-build
-  powershell_scripts: 
-    url: https://github.com/Azure/azure-docs-powershell-samples
-    branch_mapping: 
-      release-build-mysql: release-build
-      release-build-postgresql: release-build
-      release-build-stellar: release-build
-  WebApp-OpenIdConnect-DotNet: 
-    url: https://github.com/AzureADQuickStarts/WebApp-OpenIdConnect-DotNet
-    branch: GuidedSetup
+  cli_scripts: https://github.com/Azure/azure-docs-cli-python-samples
+  powershell_scripts: https://github.com/Azure/azure-docs-powershell-samples
+  WebApp-OpenIdConnect-DotNet: https://github.com/AzureADQuickStarts/WebApp-OpenIdConnect-DotNet#GuidedSetup
 ```
 Configs is grouped by scenario:
 * `contribution`: to config whether to enable contribution, and where to config.
 * `contrtibutors`: to config which contributors to include or exclude.
-* `dependencies`: to config CRRs.
+* `gitDependencies`: to config CRRs.
 Extension config is supported to share default config for all OPS repos, by `defaults: msdocs.yml`. Current config has higher priority than extension config when conflicts.
+Some configs:
+* `name`: It's equivalent to `{docset_product}.{docset_name}` in docfx v2. `product` is no long needed as a separate config in v3.
+* `include`/`exclude`: can be `string` or `string array`, indicate which file/folder to include or exclude. All contents/resources/TOCs is defined here. Document type is inferred by extension or YAMLMime. Extra config will be provided if the default inference is wrong. The latter one will overwrite the former one if matched multiple times.
+* `metadata`: means globalMeatdata if it's an object. It's array in the sample below, which can express file metadata.
 
 ### `dotnet/docs`: Complicate Metadata
 https://github.com/dotnet/docs/blob/master/docfx.json#L84-L126
 ``` yml
 metadata:
-- values:
+- path: "**"
```

Comment:
if `path` is a folder, this needs to be `/`

Reply:
**/*?

---
In reply to: [182992060](https://github.com/dotnet/docfx/pull/2644#discussion_r182992060) [](ancestors = 182992060)

Reply:
below are using folder, but here is using global?

---
In reply to: [182995408](https://github.com/dotnet/docfx/pull/2644#discussion_r182995408) [](ancestors = 182995408,182992060)